storio-test-common/src/test/java/com/pushtorefresh/storio/test/StorIOAssertTest.java

```diff
@@ -0,0 +1,39 @@
+package com.pushtorefresh.storio.test;
+
+import org.junit.Test;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import rx.Observable;
+import rx.functions.Action1;
+
+import static junit.framework.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class StorIOAssertTest {
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void assertThatObservableEmitsOncePositive() {
+        final Action1<String> testAction = mock(Action1.class);
+        final String testString = "Test string";
+        StorIOAssert.assertThatObservableEmitsOnce(Observable.just(testString), testAction);
```

Comment:
I meant to use `static import` in all usages except tests for functions that should be used via static imports :)

In test you need to understand what are you testing, with `static import` test won't be so obvious.

Your opinion?